components/org.wso2.carbon.identity.api.server.api.resource/org.wso2.carbon.identity.api.server.api.resource.v1/src/main/java/org/wso2/carbon/identity/api/server/api/resource/v1/impl/ApiResourcesApiServiceImpl.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com).
```

Comment:
What's the purpose of changing the year?

Reply:
As a practice, we change the year in the license header when we do any changes in the relevant file.

Reply:
Let's use generated year-modified year format then? Otherwise we will be loosing the information on the created year right.

Reply:
Ack